vehicle/renault.go

```diff
@@ -308,20 +313,37 @@ func (v *Renault) kamereonPerson(personID string) (string, error) {
 	return res.Accounts[0].AccountID, err
 }
 
-func (v *Renault) kamereonVehicles(accountID string) ([]string, error) {
-	uri := fmt.Sprintf("%s/commerce/v1/accounts/%s/vehicles", v.kamereon.Target, accountID)
+func (v *Renault) kamereonVehicles(configVIN string) ([]string, error) {
+	uri := fmt.Sprintf("%s/commerce/v1/accounts/%s/vehicles", v.kamereon.Target, v.accountID)
 	res, err := v.kamereonRequest(uri, nil)
-
+	var erroneousVin string
 	var vehicles []string
 	if err == nil {
-		for _, v := range res.VehicleLinks {
-			if strings.ToUpper(v.Status) == "ACTIVE" {
```

Comment:
...oder Du machst es ganz einfach und gibst hier nur aktive *UND* gepairte Fahrzeuge zurück. Dann kann ALLES andere so bleiben wie es ist...

Reply:
@andig , so richtig verstehe ich Deine Anmerkungen nicht. `kamereonVehicles` gibt auch in der master version ein string[]. Nur ist hier das string[] nicht vollständig gefiltert. Ich habe ja eigentlich genau nur das gemacht was du in dem Kommentar hier oben anmerkst nämlich die Vehicles werden zusätzlich nach  ihrer Rolle gefiltert. So das ich des weiteren ein error logging mit hinzugefügt habe ändert ja nicht an der semantic was die func zuvor zurückgegeben hat.  Der weitere Filter kam ja erst durch Deinen Kommentar beim meinem ersten Versuch hinzu. Und den finde ich auch richtig. evcc ruft diese func ja mehrfach auf für jedes vehicle was in der config hinterlegt ist und hier wird das mit eine liste von vehicles gemerged die einem kamereon account zugeordnet sind. Das Problem warum es hier falsch erscheint, ist meiner Meinung nach dass das NewRenaultFromConfig interface ein string[] verlangt. Denn die Liste der config vehicles wird nicht immer der Liste der account vehicles entsprechen. Eigentlich, durch den mehrfachen Aufruf dieser func (anzahl vehicle in der cvcc config) und die Bindung dort an genau einen account sollte die func nur immer ein vehicle zurückgeben. 

Aber ja ich könne in New.. auch eine andere func die nur die validen vehicles zurückgeben lassen und `kamereonVehicles` liefert alle account vehicles zurück , aber auch die nicht activen sonst würde man wirklich die semantic vermischen (account vehicles und in ecvv verwendbare vehicles. Also auch der bisher verwendete filter nach "ACTIVE" würde dann in die neue func wandern.